manager.java

```diff
@@ -73,46 +54,69 @@ static void ticketPrice(int totalSeats, int numOfRows, int rowNum) {
             System.out.println("Ticket price: $10");
         }
     }
-    static int numOfRows() {
+
+    static int numOfRows () {
         Scanner scanner = new Scanner(System.in);
         System.out.println("Enter the number of rows:");
         int numOfRows = scanner.nextInt();
         return numOfRows;
 
     }
-    static int rowNum() {
+    static int rowNum () {
         Scanner scanner = new Scanner(System.in);
         System.out.println("Enter a row number:");
         int rowNum = scanner.nextInt();
         return rowNum;
     }
 
-    static int numOfSeats() {
+    static int numOfSeats () {
         Scanner scanner = new Scanner(System.in);
         System.out.println("Enter the number of seats in each row:");
         int numOfSeats = scanner.nextInt();
         return numOfSeats;
     }
 
-    static int seatNum() {
+    static int seatNum () {
```

Comment:
I would try to be more consistent with your spacing. I like to use things like `static int numOfRows() {`